buildSrc/src/main/java/org/elasticsearch/gradle/internal/rest/compat/RestCompatTestTransformTask.java

```diff
@@ -105,9 +107,40 @@ public void replaceMatch(String subKey, Object value) {
         transformations.add(new ReplaceMatch(subKey, MAPPER.convertValue(value, JsonNode.class)));
     }
 
+    /**
+     * Replaces the values of a match assertion for the given REST test. For example "match":{"_type": "foo"} to "match":{"_type": "bar"}
+     *
+     * @param subKey   the key name directly under match to replace. For example "_type"
+     * @param value    the value used in the replacement. For example "bar"
+     * @param testName the testName to apply replacement
+     */
+    public void replaceMatch(String subKey, Object value, String testName) {
+        transformations.add(new ReplaceMatch(subKey, MAPPER.convertValue(value, JsonNode.class), testName));
+    }
+
+    /**
+     * A transformation to replace the key in a length assertion.
+     * @see ReplaceLength
+     * @param oldKeyName   the key name directly under length to replace.
+     * @param newKeyName   the new key name directly under length.
+     */
+    public void replaceKeyInLength(String oldKeyName, String newKeyName) {
+        transformations.add(new ReplaceLength(oldKeyName, newKeyName, null));
```

Comment:
Minor nit: what do you think about renaming `ReplaceLength` to `ReplaceKeyInLength`? The rest already match the `fooBar`/`FooBar` convention, but this one doesn't.

Reply:
563e86f